interfaces/writer_test.go

```diff
@@ -5,9 +5,12 @@ package interfaces
 
 import (
 	"bytes"
-	oldcsv "encoding/csv"
-	thiscsv "github.com/JensRantil/go-csv"
+
+	oldcsv "github.com/eltorocorp/go-csv"
```

Comment:
Similar to in the read_test.go file, this can just be `github.com/eltorocorp/go-csv'